src/masternode/utils.cpp

```diff
@@ -50,6 +50,10 @@ void CMasternodeUtils::ProcessMasternodeConnections(CConnman& connman)
             if (connman.IsMasternodeQuorumNode(pnode)) {
                 return;
             }
+            // keep _verified_ LLMQ relay connections
+            if (connman.IsMasternodeQuorumRelayMember(pnode->GetVerifiedProRegTxHash())) {
+                return;
+            }
             // keep _verified_ inbound connections
```

Comment:
@UdjinM6 that change doesn't fix the problem. We still have with rotation some mns that aren't able to relay DKG messages because they don't have verified LLMQ connections with the relay members

Reply:
Fixed bellow